### DISCUSSION TITLE
Critical - High level of failures - Gateway Timeout

### DISCUSSION THREAD

**Discussion by ShantanuNair** [deepgram/community#437] [General help]
### Which Deepgram product are you using?

Deepgram API

### Details

We are DG Partners, and are seeing a very high level of such errors:

```
DG Webhook callback body:  {
    "err_code": "Gateway Timeout",
    "err_msg": "Request timeout. Please try again later.",
    "request_id": "xxxxxxxxxxx-xxxxxxxxxxx-xxxxxxxxxxx-xxxxxxxxxxx"
}
```

A screenshot of the deepgram console logs:

<img width="623" alt="image" src="https://github.com/deepgram/community/assets/8559347/5497bff6-4bf6-40b6-a23b-cac079c9936f">

We see a lot of these metadata lost and unknown in the logs for these requests. This is critical to us

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

/v1/listen?times=true&callback=<callback>&punctuate=true&detect_topics=false&summarize=false&diarize=true&model=whisper&detect_language=true

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

<sensitive>, can share in private

**Comment by jkroll-deepgram** [comment]
Hi @ShantanuNair, I'm sorry you're encountering these Deepgram failures.

I see you're using Whisper, which is an extremely resource-intensive model and is prone to less reliable behavior. Depending on your audio languages and other requirements, I'd strongly suggest evaluating our Nova model, or at least using it in a retry strategy for periods when Whisper is not performing well.

I see we had a few periods of ~10-30 minutes over the past 24 hours in which Whisper had slightly higher error rates than normal, but all within the realm of normal operation, no incidents. If you email some affected request IDs to me, I can look further and investigate whether there are any other extenuating factors that were negatively impacting you.

**Reply by ShantanuNair** [reply]
Hi @jkroll-deepgram, some of these were with nova 1. Not all of these were with whisper. Would appreciate any and all assistance here. If we can provide any more details, privately, let me know. We want to get this resolved ASAP, as it's severely impacting us, and have to rely on fallback ASR providers due to this.

Update: Sending you an email containing request ids. Appreciate you looking into this!
